examples/prom/dg_advection_local_rom_matrix_interp.cpp

```diff
@@ -805,14 +809,11 @@ int main(int argc, char *argv[])
             std::vector<CAROM::Matrix*> M_hats;
             std::vector<CAROM::Vector*> b_hats;
             std::vector<CAROM::Vector*> u_init_hats;
-            std::ofstream fout;
-            fout.open("frequencies.txt");
```

Comment:
Why is the output of frequencies removed? Isn't this file read in some cases?

Reply:
Thanks for explaining. Maybe it is better to put the frequencies in a `CAROM::Vector` and use that class's `write` and `read` functions, for HDF files.